Sources/protoc-gen-swift/FileGenerator.swift

```diff
@@ -370,5 +370,21 @@ class FileGenerator {
             p.outdent()
             p.print("]\n")
         }
+
+        generateVersionCheck(printer: &p)
```

Comment:
Should we getting it at the top of the file so it is the first error instead of the last?